### ISSUE TITLE
Type Something

### ISSUE THREAD

**Issue by singshris** [sanctuarycomputer/chinatown.js#1]
## Chinatown.js Talk Submission

**Talk Title:** Type Something

**Talk Description:** Typography and Math are two sides of the same coin. Wonderful things are bound to happen when they join forces — concepts give rise to frameworks, static evolves into interactive, and visuals become multi-sensorial. _Type Something_ takes a closer look at how typography can be encoded with meaning beyond what meets the eye—from letterforms that follow the literal stars to symbols mapped to New York's quintessential soundscape. We anchor these processes in the people they're made for; their participation blurring the lines between designer and audience. Personable experiences become paramount in a world that treats everything like data points. Here is an approach that takes what's hidden and leverages it to drive community tools, identity systems, and playful interfaces.

**Name**: Shristi Singh

**Pronouns**: she/her

**Twitter Handle (optional)**:

**Website (optional):** [shristi.super.site](url)

**Comment by hhff** [comment]
Hi @singshris ! Your talk sounds fascinating, thank you!

QQ - will javascript actually feature in this talk, or will it be higher level? It's AOK if not, just trying to get a sense of how codey it will be <3

**Comment by singshris** [comment]
Hi @hhff, I make most of my products and art on the web, so javascript features pretty frequently but it's usually just vanilla js or the drawing libraries like p5 or matter. For this talk, I plan on giving a general overview of how javascript acts in my projects but not a line-by-line breakdown of the code itself since I find it more exciting to talk about the possibilities open-source tools bring to art and design. I'm also happy to share my presentation with you beforehand so you can give me feedback if you need me to alter the codey-ness in it. Let me know!

**Comment by hhff** [comment]
Sounds good @singshris! We'd love to accept your talk as the first speaker for July 9th. Can you please link/upload a pic here (or to ctjs@sanctuary.computer) for our promo materials?

**Comment by singshris** [comment]
![Pockets #1 Event Photos 18](https://github.com/sanctuarycomputer/chinatown.js/assets/95714281/2fd591e5-b885-4544-83e7-77499d053ab4)

This might be a suitable one! Took at at index space lol

**Comment by hhff** [comment]
wonderful thank youuuu

**Comment by singshris** [comment]
Great! Before I get the ticket for the event, I just wanted to make sure if I needed to get one? Or is [this link](https://lu.ma/5e3vxyi8) only meant for attendees?

**Comment by hhff** [comment]
Oh speakers don't require a ticket :) thank you for checking!

**Comment by hhff** [comment]
Thank you @singshris ! We'll be advertising your talk on social medias soon - looking forward to hosting you!

**Comment by singshris** [comment]
Of course! Thanks for the opportunity. I'll keep an eye out for the social media shoutout and share them too. Excited to give the talk and hear the other ones :)

**Comment by eliebelle** [comment]
Hi Shristi! Elie, Director of Index here :)

We can't wait to have you come to our space for ChinatownJS tomorrow! We'd love to invite you for a free workday at our coworking space before the event tomorrow, if you'd like.

We'll begin setting up at 6:30, so if you'd like to test your presentation, familiarize yourself with the space, and grab some food before everyone else arrives, please come early!

A few notes about the space and setup:

We ask that you keep your talk to a lightning 10 minutes, that way we have time for a few questions at the end.
-You'll be presenting using our projector's Chrome Cast, so please make sure you have a compatible browser on your machine.
Our seating area is intimate, and you'll be presenting without a microphone. That said, take this as an opportunity to practice vocal projection!
If you have any specific questions for or requirements from us, please don't hesitate to be in touch. You can reach both @hhff and me at [ctjs@sanctuary.computer](mailto:ctjs@sanctuary.computer)

Thank you so much for being part of our first ChinatownJS! We're really looking forward to meeting and hosting you.

**Comment by singshris** [comment]
Hey Elie! Sounds good. I'd love to spend some time working out of the coworking space. Planning to head post lunch, would that still be okay?

**Comment by eliebelle** [comment]
Yes absolutely! Come on over :)

You can ring the bell when you arrive!

Elie Andersen
she/her
Director

✌︎ Support our work, become a PAL <https://index-space.org/pages/join>!
☞ @index-space <https://www.instagram.com/index_space/>
✺ Index-space.org <https://www.index-space.org/>

**Comment by singshris** [comment]
Hi @eliebelle Here ya go! 
[chinatown-js-type-something.pdf](https://github.com/user-attachments/files/16354816/chinatown-js-type-something.pdf)
